debian/build_debian.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
could this have a comment saying what it does? It looks like it assumes it's running in a docker environment - could it say so?

Reply:
(and I'd prefer scripts that exist solely to be run in docker images to live in the `docker` directory)